### PR TITLE
[devtools] add WNDT63, and tags

### PR DESCRIPTION
**Pull request by kaycebasques**
I can iterate on this post, just need to land a basic draft by m63 branch point (Oct 12)

### CODE REVIEW

File: src/content/en/updates/2017/04/devtools-release-notes.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: New features and changes coming to DevTools in Chrome 59.
 
 {# wf_updated_on: 2017-04-19 #}
 {# wf_published_on: 2017-04-12 #}
-{# wf_tags: chrome59,devtools #}
+{# wf_tags: chrome59,devtools,devtools-whatsnew #}
```
Comment (petele) [inline src/content/en/updates/2017/04/devtools-release-notes.md]: Please add `devtools-whatsnew` to `src/data/commonTags.json`

Comment (kaycebasques) [reply src/content/en/updates/2017/04/devtools-release-notes.md]: Done

**Comment by kaycebasques** [comment]
Done: Images compressed

File: src/content/en/updates/2017/10/devtools-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+This issue also made it impossible to use DevTools to debug WebDriver tests.
+See the video below for an example of the issue in VS Code.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="OJlOU6UecFU"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+As of Chrome 63, this problem is no more. DevTools now supports multiple
```
Comment (Meggin) [inline src/content/en/updates/2017/10/devtools-release-notes.md]: Small nit - suggestion to lean this a little: "As of Chrome 63, DevTools now..." (remove this problem is no more and bring together two sentences).

File: src/content/en/updates/2017/10/devtools-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="tTWs0ZyXI-0"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+## Workspaces 2.0 {: #workspaces }
+
+Workspaces have been around for some time in DevTools. This feature enables
```
Comment (Meggin) [inline src/content/en/updates/2017/10/devtools-release-notes.md]: Link to Workspaces docs at the start of where you mention they've been around.

File: src/content/en/updates/2017/10/devtools-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+you to use DevTools as your IDE. You make some changes to your source code
+within DevTools, and the changes persist to the local version of your
+project on your file system.
+
+Workspaces 2.0 builds off of 1.0, adding a more helpful UX and improved
```
Comment (Meggin) [inline src/content/en/updates/2017/10/devtools-release-notes.md]: General comment that this section is a bit 'thisy'. Suggestion to remove some of the thises.

File: src/content/en/updates/2017/10/devtools-release-notes.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+auto-mapping of transpiled code.
+This feature was originally scheduled to be released shortly after
```
Comment (Meggin) [inline src/content/en/updates/2017/10/devtools-release-notes.md]: Is it necessary to explain how the feature shifted in being released? It is somewhat distracting from the feature. Suggestion to delete this sentence, which would lean the text, and get rid of 'thisiness'.

File: src/content/en/updates/2017/10/devtools-release-notes.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+CDS 2016 to see Workspaces 2.0 in action.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="HF1luRD4Qmk"
+          data-autohide="1" data-showinfo="0" frameborder="0" 
+          data-start="868" allowfullscreen>
+  </iframe>
+</div>
+
+## Simulate push notifications with custom data {: #push }
```
Comment (Meggin) [inline src/content/en/updates/2017/10/devtools-release-notes.md]: Oh this feature is sweet!

File: src/content/en/updates/2017/10/devtools-release-notes.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+
+Simulating push notifications has been around for a while in DevTools,
+with one limitation: you couldn't send custom data. But with
+the new **Push** text box coming to the **Service Worker** pane in Chrome 63,
+now you can. Try it now:
```
Comment (Meggin) [inline src/content/en/updates/2017/10/devtools-release-notes.md]: Reads a little funny. Following a similar style to background sync events below, how about something like this:

"Simulating push notifications has been around for awhile in DevTools, but you couldn't send custom data. The new push text box coming to the Service Worker pane in Chrome 63 lets you write something in a text box and send it as custom data:"

File: src/content/en/updates/2017/10/devtools-release-notes.md [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+See the video below for an example of the issue in VS Code.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="OJlOU6UecFU"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+As of Chrome 63, this problem is no more. DevTools now supports multiple
+remote debugging clients. Watch the video below to see an example of VS Code
```
Comment (paulirish) [inline src/content/en/updates/2017/10/devtools-release-notes.md]: A few more usecases that work now:

* A DevTools Protocol client connected to the `--remote-debugging-port` websocket (e.g. chrome-remote-interface) and an open DevTools window
  * Lighthouse or Puppeteer plus DevTools
  * vscode-chrome-debug or WebStorm Chrome Debugging plus DevTools
* Two websocket protocol clients at once. (Two puppeteer sessions connecting to the same tab)
* `chrome.debugger` extension API plus DevTools
* Two different chrome extensions both using `chrome.debugger`


Unrelated but powerful together: `chrome.debugger` is now available from within a devtools extension's `devtools_page`. (New as of like 6mo ago)

----------

I'd love to have the above usecases included in this blog post.

Also we might as well call out this was devtools #1 bug and chromium's #3 bug.

**Comment by kaycebasques** [comment]
@paulirish I mentioned the bugs and use cases in my latest commit. didn't mention the `chrome.debugger` being available from devtools extensions, though, because I couldn't make it relevant to the multi-client story

**Comment by kaycebasques** [comment]
Dismissing paul's and pete's reviews to get this landed before branch point and make sure that the devtools UI integration is OK. I think I handled all of their feedback anyway. I can subsequently update the post if needed.

**Pull request merged**